### PR TITLE
Completed node-module-system exercise. - Tiffany Brown

### PR DESCRIPTION
**Pull request by tiffanydbrown**
Screenshot 2023-12-12 at 11 08 52 AM

### CODE REVIEW

File: node-module-system/calculate.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const divideOperand = divide(Number(process.argv[4]), Number(process.argv[2]));
+const multiplyOperand = multiply(
+  Number(process.argv[2]),
+  Number(process.argv[4])
+);
+
+console.log('result:', addOperand);
+console.log('result:', minusOperand);
+console.log('result:', divideOperand);
+console.log('result:', multiplyOperand);
```
Comment (RobertGardner) [inline node-module-system/calculate.ts:17]: This is very close, but you should only do the operation that was specified by the second command line argument. So you need to get that argument and switch on it, only performing the operation it specifies.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
It doesn't quite do the right thing yet. See comment.

**Comment by tiffanydbrown** [comment]
Screenshot 2023-12-12 at 11 27 53 AM

File: node-module-system/calculate.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    result = subtract(num2, num1);
+    break;
+  case 'times':
+    result = multiply(num1, num2);
+    break;
+  case 'over':
+    result = divide(num2, num1);
+    break;
+}
+console.log(result);
```
Comment (RobertGardner) [inline node-module-system/calculate.ts:25]: Nice solution! There's no code duplication I can find, but some error handling would be nice.

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**